platform-controller/src/main/java/org/hobbit/controller/config/HobbitConfig.java

```diff
@@ -0,0 +1,43 @@
+package org.hobbit.controller.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Created by Tim Ermilov on 15.05.17.
+ */
+public class HobbitConfig {
+    public class TimeoutConfig {
+        public long benchmarkTimeout;
+        public long challengeTimeout;
+
+        public TimeoutConfig(long bt, long ct) {
+            benchmarkTimeout = bt;
+            challengeTimeout = ct;
+        }
+    }
+
+    public Map<String, LinkedHashMap<String, Object>> timeouts;
+
+    public static HobbitConfig loadConfig() throws IOException {
+        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+        String file = HobbitConfig.class.getClassLoader().getResource("config.yaml").getFile();
```

Comment:
I am very sure that this will end with a `NullPointerException` in many cases.
1. `getResource` returns `null` if the resource couldn't be found, e.g., if the user didn't mounted the file.
2. I think we have to define a default path for this file that the user simply has to use or give an env variable key with which the user can define the file. Loading something as resource does not mean that the JVM will search through everything searching for a file with the given name.
3. If you want to stick to using Resources, I would like to point out that loading a resource and trying to read it as a file is a bad combination in some cases since not all resources are available as files. You might want to check if you can give an `InputStream` object to the `mapper` instead of the `File` object. That would enable the usage of `getResourceAsStream`.